src/components/settings/Settings.h

```diff
@@ -106,7 +139,7 @@ namespace Pinetime {
         uint32_t stepsGoal = 10000;
         uint32_t screenTimeOut = 15000;
 
-        WakeUpMode wakeUpMode = WakeUpMode::None;
+        std::bitset<4> wakeUpMode {static_cast<size_t>(WakeUpMode::None)};
```

Comment:
This doesn't initialize the `bitset` with bit 0 set and all other bits cleared which I think was the intended behaviour. It initializes the `bitset` with the 4 least significant bit of the value 0, so all bits start cleared.